website/content/docs/import/gcpsm.mdx

```diff
@@ -0,0 +1,53 @@
+---
+layout: docs
+page_title: Google Cloud Platform Secret Manager - Secrets Import Source
+description: The Google Cloud Platform Secret Manager source imports secrets from GCP to Vault.
+---
+
+# GCP import source
+
+
+Use the GCP source to import secret data from GCP Secret Manager into your Vault instance. To use dynamic
```

Comment:
```suggestion
Use the GCP source to import secret data from GCP secret manager into your Vault instance. To use dynamic
```

Reply:
GCP Secret Manager is the name of the product from Google, should it stay capitalized for this?